samples/v2/pipeline_with_importer_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2021 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Hello world v2 engine pipeline."""
+
+from __future__ import annotations
+
+import unittest
+from pprint import pprint
+
+import kfp
+import kfp_server_api
+from ml_metadata.proto import Execution
+
+from .pipeline_with_importer import pipeline_with_importer
+from ..test.util import KfpTask, KfpArtifact, TaskInputs, TaskOutputs, run_pipeline_func, TestCase, KfpMlmdClient
+
+
+def verify(run: kfp_server_api.ApiRun, mlmd_connection_config, **kwargs):
+    t = unittest.TestCase()
+    t.maxDiff = None  # we always want to see full diff
+    t.assertEqual(run.status, 'Succeeded')
+    client = KfpMlmdClient(mlmd_connection_config=mlmd_connection_config)
+    tasks = client.get_tasks(run_id=run.id)
+    task_names = [*tasks.keys().sort()]
```

Comment:
https://github.com/kubeflow/pipelines/blob/5203ecf273951fb1cd69f90e23c816e340e41372/samples/test/metrics_visualization_v2_test.py#L34

We are using assert count equal which is not order sensitive

Reply:
Many tests are using assertEqual instead of assertCountEqual like https://github.com/kubeflow/pipelines/blob/5203ecf273951fb1cd69f90e23c816e340e41372/samples/test/two_step_with_uri_placeholder_test.py#L30 and https://github.com/kubeflow/pipelines/blob/5203ecf273951fb1cd69f90e23c816e340e41372/samples/test/two_step_test.py#L32

Reply:
I will also fix them in this PR